www/front_src/src/Login/models.ts

```diff
@@ -23,3 +23,11 @@ export interface PlatformVersions {
     version: string;
   };
 }
+
+export interface ProviderConfiguration {
+  authenticationUri: string;
+  id: number;
+  isActive: boolean;
+  isForced?: boolean;
```

Comment:
Why the isForced is not set with a default value on this feature ?

Reply:
Because the backend as a very default configuration stored in the database. By default, the isForce is `undefined`which means false